### ISSUE TITLE
Second tool not showen

### ISSUE THREAD

**Issue by EddyMI3d** [jneilliii/OctoPrint-PlotlyTempGraph#20]
Hello @jneilliii !

With the latest update (V 0.1.2), the second tool does not appear in the graph any more.

![grafik](https://user-images.githubusercontent.com/8533327/133923500-71e0a0aa-dbf9-4089-aebb-550f1b7e3afd.png)

**Comment by jneilliii** [comment]
Can you try adding the second tool to the name map?

**Comment by aerickson** [comment]
I thought that this was a bug also. 

It's hard to figure out the additional temperature values (I was missing 'actual'... in my plugin I don't add this). I think the previous behavior of displaying all is better/less confusing (with users adding a config entry if hiding a value is desired).

Is there a way to inspect the temperature names in the javascript console?

Thanks for all of your work. :)

**Comment by jneilliii** [comment]
I agree, this is less than ideal but number of users with multiple extruders is probably not that high. I will plan on figuring out a way to automatically pick up what is returned from the printer and add to the name map though.

**Comment by EddyMI3d** [comment]
> 
> 
> Can you try adding the second tool to the name map?

Yes, that worked for me too. Thanks you

**Comment by jneilliii** [comment]
Adding note, that this also effects the Enclosure Plugin fork as sensors aren't automatically added to the graph anymore with the name map. Will be digging into this further over the weekend to remedy.
